src/components/Page/Body/CardList/CardItem/index.js

```diff
@@ -0,0 +1,32 @@
+// import PropTypes from 'prop-types';
+import * as Styled from './style.js';
+import PropTypes from 'prop-types';
+import { Card } from '../../../../Card';
+
+/**
+ * Primary UI component for user interaction
+ */
+
+export const CardItem = ({ card }) => {
+  const { company, numbers, owner, validDay, nickName } = card;
+  return (
+    <>
+      <Styled.Container>
+        <Card
+          size={'small'}
+          company={company}
+          numbers={numbers}
+          owner={owner}
+          validDay={validDay}
+        />
```

Comment:
```suggestion
export const CardItem = ({ card: { nickName, ...cardProps }) => {
  return (
    <>
      <Styled.Container>
        <Card
          size={'small'}
          {...cardProps}
        />
```

불필요한 destructuring을 이렇게도 줄여볼 수 있을 것 같네요

Reply:
하지만 이렇게 하기 전에 props drilling이 발생하고 있는 이 상황이 정상인가 먼저 고민해보셔야 할 것 같아요~